### PR TITLE
[bitnami/phpmyadmin] Chart standardized

### PR DESCRIPTION
**Pull request by migruiz4**
**Description of the change**

This PR standardizes the bitnami/phpmyadmin chart to be in line with the rest of the catalog.

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/phpmyadmin/values.yaml [hunk 1]
```diff
@@ -311,15 +356,32 @@ service:
...
+
 ## Ingress configuration
 ##
 ingress:
   ## @param ingress.enabled Set to true to enable ingress record generation
   ##
   enabled: false
+  ## @param ingress.apiVersion Force Ingress API version (automatically detected if not set)
+  ##
+  apiVersion:
```
Comment (pablogalegoc) [inline bitnami/phpmyadmin/values.yaml:384]:
```suggestion
  apiVersion: ""
```

File: bitnami/phpmyadmin/values.yaml [hunk 2]
```diff
@@ -277,12 +322,12 @@ service:
   ## @param service.type Kubernetes Service type
   ##
   type: ClusterIP
-  ## @param service.port Service HTTP port
+  ## @param service.ports Service HTTP port
```
Comment (pablogalegoc) [inline bitnami/phpmyadmin/values.yaml:325]:
```suggestion
  ## @param service.ports.http Service HTTP port
```

File: bitnami/phpmyadmin/templates/deployment.yaml [hunk 3]
```diff
@@ -43,6 +43,14 @@ spec:
...
         nodeAffinity: {{- include "common.affinities.nodes" (dict "type" .Values.nodeAffinityPreset.type "key" .Values.nodeAffinityPreset.key "values" .Values.nodeAffinityPreset.values) | nindent 10 }}
       {{- end }}
+      {{- if .Values.schedulerName }}
+      schedulerName: {{ .Values.schedulerName }}
+      {{- end }}
+      {{- if .Values.topologySpreadConstraints }}
+      topologySpreadConstraints: {{- include "common.tplvalues.render" (dict "value" .Values.topologySpreadConstraints "context" .) | nindent 8 }}
+      {{- end }}
+      priorityClassName: {{ .Values.priorityClassName | quote }}
+      {{- if .Values.podSecurityContext.enabled }}
```
Comment (pablogalegoc) [inline bitnami/phpmyadmin/templates/deployment.yaml:53]: Missing the `securityContext` here

File: bitnami/phpmyadmin/values.yaml [hunk 4]
```diff
@@ -473,10 +553,37 @@ metrics:
     annotations:
       prometheus.io/scrape: "true"
       prometheus.io/port: "{{ .Values.metrics.service.port }}"
+    ## @param service.clusterIP phpmyadmin service Cluster IP
```
Comment (pablogalegoc) [inline bitnami/phpmyadmin/values.yaml:556]:
```suggestion
    ## @param metrics.service.clusterIP phpmyadmin service Cluster IP
```
And the parameters below also need the `metrics.` at the begining

**Review by pablogalegoc — COMMENTED** [review]
Thanks @migruiz4! I left some comments below

File: bitnami/phpmyadmin/templates/deployment.yaml (hunk 3 above)
Comment (migruiz4) [reply bitnami/phpmyadmin/templates/deployment.yaml:53]: Thank you, this line was a typo. Fixed.

**Review by marcosbc — APPROVED** [review]
LGTM

File: bitnami/phpmyadmin/values.yaml [hunk 5]
```diff
@@ -479,10 +560,37 @@ metrics:
...
+    ## @param metrics.service.sessionAffinity Session Affinity for Kubernetes service, can be "None" or "ClientIP"
+    ## If "ClientIP", consecutive client requests will be directed to the same Pod
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies
+    ##
+    sessionAffinity: None
+    ## @param metrics.service.sessionAffinityConfig Additional settings for the sessionAffinity
+    ## sessionAffinityConfig:
+    ##   clientIP:
+    ##     timeoutSeconds: 300
+    sessionAffinityConfig: {}
```
Comment (carrodher) [inline bitnami/phpmyadmin/values.yaml:592]:
```suggestion
    ##     timeoutSeconds: 300
    ##
    sessionAffinityConfig: {}
```

**Pull request merged**